packages/loot-core/src/platform/server/fetch/index.web.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-restricted-globals */
+// (the `self` global is conventional in web workers)
```

Comment:
same here: would be nice to define it as a global once instead of disabling the rule in each file individually